src/screens/ProfileScreen/index.js

```diff
@@ -629,23 +629,25 @@ const ProfileScreen = ({route}) => {
   };
 
   const onHeaderOptionClicked = (post) => {
+    console.log('post', post);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not a good practice to leave console.log statements in the production code. It can expose potentially sensitive user data and make debugging harder.
```diff
- console.log('post', post);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ada 2 console belum ke hapus mas